dipy/workflows/multi_io.py

```diff
@@ -253,3 +253,9 @@ def __iter__(self):
         IO = np.concatenate([I, O], axis=1)
         for i_o in IO:
             yield i_o
+
+    def file_existence_check(self, args):
+        input_args = list(args)
```

Comment:
Thank you for this update @parichit. 

As you can see [here](https://travis-ci.org/nipy/dipy/jobs/424453062#L3493), the tests are still failing. I think you can replace this line with something like:
`input_args =  [fname for fname in list(args) if isinstance(fname, str)]`

it allow to keep only string value and not integers. Indeed, we can have integers as positional argument sometimes

Reply:
Thanks for this suggestion @skoudoro , I wasn't sure why the PR was failing even after I made the change. You are right, there were other arguments type than string. I changed the line to accept only strings. Waiting for the checks to complete.